plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/editor/ERDEditorEmbedded.java

```diff
@@ -50,9 +52,12 @@
 import org.jkiss.utils.CommonUtils;
 import org.jkiss.utils.xml.XMLUtils;
 import org.w3c.dom.Document;
-
+import org.jkiss.dbeaver.utils.GeneralUtils;
+import org.jkiss.dbeaver.erd.ui.action.DiagramExportAction;
```

Comment:
Remove unused imports

Reply:
import org.jkiss.dbeaver.erd.ui.action.DiagramExportAction; needs to be in ERDEditorEmbedded.java in order to generate the button on the toolBarManager